src/App.jsx

```diff
@@ -1,25 +1,24 @@
 import React, { Component } from "react";
 import axios from "axios";
-
 class App extends Component {
   state = {
     query: "",
     results: [],
-    message: ""
+    message: "",
+    details: [],
+    details_error_message: "",
```

Comment:
1. Do we need a specific state for this message 2. Please use camelCase in JavaScript

Reply:
Removed this as well since we weren't actually using it